site/app/models/GradingSection.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace app\models;
+
+use app\libraries\Core;
+
+/**
+ * Represents a single section of students or teams, and their graders
+ * Can be either registration or rotating
+ *
+ * @package app\models
+ * @method Gradeable getGradeable()
+ * @method string getName()
+ * @method User[] getGraders()
+ * @method User[] getUsers()
+ * @method Team[] getTeams()
+ */
+class GradingSection extends AbstractModel {
+    /**
+     * @var Gradeable
+     */
+    protected $gradeable;
```

Comment:
Does this class really need a reference to the `Gradeable` instance?

Reply:
No, it really doesn't.